### PR TITLE
Fix the link

### PR DESCRIPTION
**Pull request by banyan**

### CODE REVIEW

**Comment by goloroden** [comment]
Thanks a lot for this PR 😊

It’s a good catch, the link has actually been wrong. Unfortunately, the fix points to the wrong page. It should be: https://docs.wolkenkit.io/latest/getting-started/creating-your-first-application/setting-the-objective/

Could you please change that and add yourself to the list of contributors in package.json?

That would be great 😊

**Comment by banyan** [comment]
oops, I've just fixed 😄

**Comment by goloroden** [comment]
Hi @banyan :-)

Maybe something went wrong while pushing (I've seen that you `force` pushed), since the updates are not shown to me (neither the link, nor the update to the `package.json` file).

Sorry for asking again, but could you please check this?

(Oh, and no need to `force` push – we are going to squash the PR anyway, so just push additional changes 😉)

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ $ npm install -g wolkenkit
 
 ## Quick start
 
-First, have a look at the [documentation](https://docs.wolkenkit.io), and its getting started [guides](https://docs.wolkenkit.io/latest/guides/creating-your-first-application/setting-the-objective/). You should also learn about [why to use wolkenkit](https://docs.wolkenkit.io/latest/getting-started/understanding-wolkenkit/why-wolkenkit/).
+First, have a look at the [documentation](https://docs.wolkenkit.io), and its getting started [guides](https://docs.wolkenkit.io/latest/guides/creating-an-application-from-scratch/setting-the-objective/). You should also learn about [why to use wolkenkit](https://docs.wolkenkit.io/latest/getting-started/understanding-wolkenkit/why-wolkenkit/).
```
Comment (goloroden) [inline README.md]: The fixed link should be https://docs.wolkenkit.io/latest/getting-started/creating-your-first-application/setting-the-objective/

**Comment by banyan** [comment]
Sorry for the trouble 😓

**Comment by goloroden** [comment]
Thanks 😊

**Comment by goloroden** [comment]
PS: Everything is fine, and thanks for taking care 😊

**Pull request merged**